### ISSUE TITLE
"deprecated Object#=~ is called on Array; it always returns nil" written in the standard error a lot of times

### ISSUE THREAD

**Issue by MarcoCostantini** [flyerhzm/rails_best_practices#378]
I tried rails_best_practices (version 1.20.0) on my file, but it wrote in the standard error, 2674 times, the line
```
/var/lib/gems/2.7.0/gems/rails_best_practices-1.20.0/lib/rails_best_practices/core/check.rb:44: warning: deprecated Object#=~ is called on Array; it always returns nil
```

**Comment by ybiquitous** [comment]
@MarcoCostantini I believe this issue was fixed by PR #370 (merged to the master branch).

See also #353

**Comment by alex-benoit** [comment]
@flyerhzm Would it be possible to release a version on rubygems with this fix?
